lumigator/frontend/src/components/molecules/LDatasetEmpty.vue

```diff
@@ -1,10 +1,15 @@
 <template>
   <div class="l-dataset-empty">
-    <h2 class="l-dataset-empty__instructions">Submit a dataset to start using Lumigator.</h2>
+    <h2 class="l-dataset-empty__instructions">Use a dataset as the basis for your evaluation. It includes data for the model you'd like to
+      evaluate and possibly a ground truth "answer".</h2>
     <p class="l-dataset-empty__instructions-text">
       <span>
-        Use a dataset as the basis for your evaluation. It includes data for the model you'd like to
-        evaluate and possibly a ground truth "answer".
+        If you don't have your dataset yet, you can use our
+        <a
+        href="https://github.com/mozilla-ai/lumigator/tree/7b3112a40d44c671810453961a8e8f6fbb0b18c3/lumigator/sample_data"
```

Comment:
Would this be better?
```suggestion
        href="https://github.com/mozilla-ai/lumigator/tree/main/lumigator/sample_data"
```

Reply:
ah yes!